todo/ios/MongoDBSample/Classes/Controllers/TodoList/TodoListViewController.swift

```diff
@@ -7,14 +7,23 @@ import UIKit
 import FacebookLogin
 import FacebookCore
 import StitchCore
-import ExtendedJson
-import MongoDBService
+import MongoSwiftMobile
+import StitchRemoteMongoDBService
 
-
-class TodoListViewController: UIViewController, UIStitchDelegate, AuthenticationViewControllerDelegate, EmailAuthViewControllerDelegate, TodoItemTableViewCellDelegate, UITableViewDataSource {
+class TodoListViewController: UIViewController, AuthenticationViewControllerDelegate, EmailAuthViewControllerDelegate, TodoItemTableViewCellDelegate, UITableViewDataSource {
+    
+    private struct Consts {
+        static var AppId: String {
+            let path = Bundle.main.path(forResource: "Stitch-Info", ofType: "plist")
+            let infoDic = NSDictionary(contentsOfFile: path!) as? [String: AnyObject]
+            let appId = infoDic!["APP_ID"] as! String
+            assert(appId != "<Your-App-ID>", "Insert your App ID in Stitch-Info.plist")
+            return appId
+        }
+    }
     
-    private var stitchClient : StitchClient?
-    private var mongoClient: MongoDBClient?
+    private var stitchClient : StitchAppClient?
```

Comment:
the rest of the code will look cleaner if you make these implicitly unwrapped

```swift
private var stitchClient: StitchAppClient!
private var mongoClient: RemoteMongoClient!
```

Reply:
Done